Compile error when building

Hi! Very cool project :-)

I'm getting a compile error when trying to build it on Mac OSX

```
$ make with_brew
cc -g3 -Wall -I/usr/local/include -I/usr/local/include/editline -O0 -std=gnu99 -c builtins.c
builtins.c:33:11: error: use of undeclared identifier 'u8'
        { u8"λ",          Builtin_lambda },
          ^
1 error generated.
make: *** [builtins.o] Error 1
```

Apparently my compiler doesn't like the `u8` prefix. I did a quick search and it seems to be a valid prefix on C++ but could not find it in the docs for C.

Cheers!

--- a/builtins.c
+++ b/builtins.c
@@ -30,7 +30,7 @@ Builtin_init(void)
         { "def",          Builtin_def },
         { "cons",         Builtin_cons },
         { "list",         Builtin_list },
-        { u8"λ",          Builtin_lambda },
+        { "\xce\xbb",     Builtin_lambda },
         { "lambda",       Builtin_lambda },
         { "macroexpand-1",Builtin_macroexpand_1 },
         { "macroexpand",  Builtin_macroexpand },

